dstrf/_model.py

```diff
@@ -844,7 +846,7 @@ def eval_obj(self, data):
         -------
             float
         """
-        v = 0
+        residual = 0
```

Comment:
Maybe there could be better variables names (would be good also for Return part of the docs) I just used what the callers used

Reply:
I was just being lazy. :)